File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    answer = []
    for i in numbers:
        answer.append(i)
    if direction == 'right':
        for i in range(len(numbers)):
            answer[i] = numbers[i-1]
    elif direction == 'left':
        for i in range(len(numbers)):
            if i+1 == len(numbers):
                answer[i] = numbers[0]
            else:
                answer[i] = numbers[i+1]
    return answer

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
class Solution {
    public int[] solution(int[] numbers, String direction) {
        int len = numbers.length;
        int[] answer = new int[len];
        if (direction.equals("right")) {
            for (int i = 0; i < len; i++) {
                answer[(i + 1) % len] = numbers[i];
            }
        } else {
            for (int i = 0; i < len; i++) {
                answer[(i + len - 1) % len] = numbers[i];
            }
        }
        return answer;
    }
}